data/yaml/name_variants.yaml

```diff
@@ -5460,6 +5460,9 @@
 - canonical: {first: Yang, last: Liu}
   comment: Edinburgh
   id: yang-liu-edinburgh
+- canonical: {first: Yang, last: Liu}
```

Comment:
Are we sure this is a new Yang Liu? If unsure, I would just use the name without an "id", for later sorting.

Reply:
That's not possible, is it? All instances of ambiguous names must be disambiguated, or we raise an error.

Reply:
Correct, thanks for clarifying. You'd have to use the ID of the "base class", e.g., `id="yang-liu".

Reply:
The institution associated with this author is new.